litewallet/Platform/Extensions.swift

```diff
@@ -558,3 +558,17 @@ extension Dictionary where Key: ExpressibleByStringLiteral, Value: Any {
 		return jstring
 	}
 }
+
+extension UInt64 {
+	/// Performs a safe addition, returning `nil` if overflow occurs.
```

Comment:
While the existing documentation is clear, it can include more details about when these methods are useful.
/// Adds two `UInt64` values safely, returning `nil` if overflow occurs.
/// - Parameter value: The value to add.
/// - Returns: The sum of the two values or `nil` if the operation overflows.